src/styles/utils/_animations.scss

```diff
@@ -0,0 +1,4 @@
+%easyIn {
```

Comment:
If u'll use SRP, figure out how to name it will be little easy

https://en.wikipedia.org/wiki/Single-responsibility_principle